src/utils/resources.py

```diff
@@ -1106,13 +1105,25 @@ def __init__(self, properties: Dict[str, Any], *args, **kwargs):
             if isinstance(values.get("packages"), str):
                 values["packages"] = [value.strip() for value in values["packages"].split(",")]  # type: ignore
 
+            if isinstance(values.get("packages_from_raw_bash"), str):
+                out, err = self.check_output_bash_snippet(values.get("packages_from_raw_bash"))
+                if err:
+                    logger.error(
+                        "Error while running apt resource packages_from_raw_bash snippet for '" + str(key) + "' extras:"
+                    )
+                    logger.error(err)
+                values["packages"] = [value.strip() for value in out.split("\n")]
+
             if (
                 not isinstance(values.get("repo"), str)
                 or not isinstance(values.get("key"), str)
-                or not isinstance(values.get("packages"), list)
+                or (
+                    not isinstance(values.get("packages"), list)
+                    and not isinstance(values.get("packages_from_raw_bash"), str)
```

Comment:
I'm not sure that bit is needed ? If `packages_from_raw_bash` is defined, then `packages` should be defined as well because of the previous code ?